mappings/net/minecraft/client/render/OverlayTexture.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_4608 net/minecraft/client/render/OverlayTexture
+	FIELD field_21013 texture Lnet/minecraft/class_1043;
+	METHOD method_23209 setupOverlayColor ()V
+	METHOD method_23210 getU (F)I
+	METHOD method_23211 clearDefaultOverlay (Lnet/minecraft/class_4588;)V
+		ARG 0 vertexConsumer
+	METHOD method_23212 getV (Z)I
```

Comment:
:thinking: why does `getU` take a float while `getV` takes a boolean

Reply:
Not sure what the parameters mean yet, but it's just the way the overlay texture is laid out. It's a grid of overlays with two columns and many rows. The row number depends on something, and the column depends on something else.